requirements.txt

```diff
@@ -1,16 +1,15 @@
-certifi==2022.6.15
-duneapi==8.0.0
-dune-client==1.1.1
+certifi>=2022.6.15
+duneapi>=8.0.0
+dune-client>=1.1.1
 psycopg2-binary>=2.9.6
 python-dotenv>=0.20.0
 coinpaprika>=0.1.0
 requests>=2.28.1
-safe-cli==0.7.0
-safe-eth-py==5.5.0
-slackclient==2.9.4
-web3>=6.20.0
+safe-eth-py>=5.5.0
+slackclient>=2.9.4
+web3<7.0.0
```

Comment:
What was wrong with 7? I updated other code recently and didn't have to make any changes to its usage.

Reply:
I get an error from `gnosis/eth/ethereum_client.py:41:` where the deprecated name `geth_poa_middleware` is used instead of the new name `ExtraDataToPOAMiddleware`. Either the new web3 is not supported yet or one has to force it so some newer version. I would have expected a clean install with few dependencies would automatically use the most recent version.